service-to-worker/README.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: pattern
+title: Service to Worker
+folder: service-to-worker
+permalink: /patterns/service-to-worker/
+categories: Architectural
+tags: - Decoupling
+---
+
+## Intent
+
+Combine a controller and dispatcher with views and helpers to handle client requests and prepare a dynamic presentation as the response. Controllers delegate content retrieval to helpers, which manage the population of the intermediate model for the view. A dispatcher is responsible for view management and navigation and can be encapsulated either within a controller or a separate component.
+
+## Explanation
+The system controls the execution flow and access flow of business data, creates presentation content from it, and centralizes control and improves modularity, reuse, and separation of roles.
+
+
+**Example**
+
+We modified this pattern based on a classic design patterns [Model View Controller Pattern](https://github.com/iluwatar/java-design-patterns/tree/master/model-view-controller) as the Class Diagram.
+
+In this example, the controller directly controls the display of the View and can receive commands to control the dispatcher indirectly. The dispatcher stores different commands that can be used to modify the `model` with `action`s or to modify the display in the `view`s.
+
+Therefore, this example leverages the Service to Worker pattern to increase functionality cohesion and improve the business logic.
```

Comment:
Please use the same explanation format as we do in other patterns e.g. https://github.com/iluwatar/java-design-patterns/tree/master/abstract-factory